nodebuilder/core/config.go

```diff
@@ -16,9 +16,10 @@ var MetricsEnabled bool
 
 // Config combines all configuration fields for managing the relationship with a Core node.
 type Config struct {
-	IP       string
-	RPCPort  string
-	GRPCPort string
+	IP        string
+	RPCPort   string
+	GRPCPort  string
+	EnableTLS bool
```

Comment:
let's avoid growing the node's config. We have an env variable that signals that TLS should be enabled.